frontend/templates/webhooks_dashboard.html

```
@@ -44,9 +44,9 @@ <h2 class="text-base font-semibold text-gray-900 dark:text-white">Configured end
                     <h3 class="truncate text-sm font-semibold text-gray-900 dark:text-white" x-text="hook.description || hook.url"></h3>
                     <span class="rounded px-2 py-0.5 text-xs font-medium"
                           :class="hook.is_active ? 'bg-green-100 text-green-800 dark:bg-green-900 dark:text-green-200' : 'bg-gray-100 text-gray-700 dark:bg-gray-700 dark:text-gray-200'"
-                          x-text="hook.is_active ? 'Active' : 'Paused'"></span>
+                          x-text="hook.is_active ? '{{ _("webhooks.active") }}' : '{{ _("webhooks.paused") }}'"></span>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)